AI/modules/signal/models/patchtst/train.py

```
@@ -330,9 +354,18 @@ def train():
     with open(scaler_path, 'wb') as f:
         pickle.dump(scaler, f)
 
+    metadata = build_patchtst_metadata(
+        config=CONFIG,
+        model_path=model_path,
+        scaler_path=scaler_path,
+        feature_columns=FEATURE_COLUMNS,
+    )
+    save_patchtst_metadata(metadata_path, metadata)
+
     print(f"\n>> 완료")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)